src/player.cpp

```diff
@@ -157,7 +157,7 @@ static const itype_id OPTICAL_CLOAK_ITEM_ID( "optical_cloak" );
 static bool should_combine_bps( const player &, size_t, size_t );
 
 
-player::player() : Character()
```

Comment:
Any particular reason to change this?  The same invocation happens implicitly, I'd prefer that it be explicit.